Override braces to ^3.0.3.

@generated
@generator: scripts/apply-override.sh --patch common/override-braces.diff --override 'global:braces=^3.0.3'
@override-package: braces@^3.0.3

Index: code-editor-src/package.json
===================================================================
--- code-editor-src.orig/package.json
+++ code-editor-src/package.json
@@ -236,6 +236,7 @@
     "yaserver": "^0.4.0"
   },
   "overrides": {
+    "braces": "^3.0.3",
     "node-gyp-build": "4.8.1",
     "serialize-javascript": "^7.0.3"
   },
